Paymetheus/Request.xaml

```diff
@@ -7,7 +7,7 @@
       xmlns:vm="clr-namespace:Paymetheus.ViewModels"
       mc:Ignorable="d"
       x:Name="Page"
-      WindowTitle="Page"
+      WindowTitle="Request"
```

Comment:
looks like an unnecessary change, please revert it unless there was a reason for it